ui/src/routes.ts

```diff
@@ -1,11 +1,11 @@
 export type RouteID =
   | "connectionDetails"
   | "connections"
-  | "credentialLink"
   | "credentials"
   | "importSchema"
   | "issueCredential"
   | "notFound"
+  | "scanLink"
```

Comment:
This is not an expressive route IMO. It should follow the convention of referencing the domain. E.g. `credentialQR` or `scanCredential`.